spec/sidekiq/jobs/inning_rollover_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.inline!
+
+RSpec.describe InningRolloverJob, type: :job do
+  before :each do
+    Sidekiq::Testing.fake!
+    @inning1 = create(:inning, :is_current)
+    @inning2 = create(:inning, :not_current_future)
+  end
+
+  describe 'queueing' do
+    it 'queues the job' do
+      expect { InningRolloverJob.perform_async(@inning2.id) }.to change(InningRolloverJob.jobs, :size).by(1)
+      Sidekiq::Worker.drain_all
+    end
+  end
+
+  describe 'executes perform' do
+    it 'makes the new inning current and the old inning not current' do
+      InningRolloverJob.perform_async(@inning2.id)
+    
+      Sidekiq::Worker.drain_all
+
```

Comment:
There are a lot of instances in your tests where you call `perform_async` and then `drain_all` to perform that job immediately. This works, but I think it would be more elegant to set the testing mode to `inline` which makes jobs perform immediately rather than queueing them.

Reply:
fixed!